docs/reference/components/sinks/syslog.cue

```diff
@@ -0,0 +1,244 @@
+			required:    false
+			warnings: []
+			type: bool: default: false
+		}
+		appname_key: {
+			common:      false
+			description: "The key for the field that will be used as the `APP-NAME`."
+			required:    false
+			warnings: []
+			type: string: {
+				default: "appname"
+				examples: ["application_name"]
+				syntax: "literal"
+			}
+		}
+		facility_key: {
+			common: false
+			description: """
+				The key for the field that will be used as the Syslog facility. The following value will be recognized as
+				valid facility: integer from 0 (included) to 23 (included) and the matching string representation: `kern`, `user`,
+				`mail`, `daemon`, `auth`, `syslog`, `lpr`, `news`, `uucp`, `cron`, `authpriv`, `ftp`, `ntp`, `audit`, `alert`, `clockd`,
+				`local0`, `local1`, `local2`, `local3`, `local4`, `local5`, `local6` and `local7`. Please check the
+				[RFC5424](https://tools.ietf.org/html/rfc5424) for additional information.
+				"""
+			required: false
+			warnings: []
+			type: string: {
+				default: "facility"
+				examples: ["facility"]
+				syntax: "literal"
+			}
+		}
+		host_key: {
+			common:      false
+			description: "The key for the field that will be used to as the host in Syslog messages. This overrides the [global `host_key` option][docs.reference.configuration.global-options#host_key]."
+			required:    false
+			warnings: []
+			type: string: {
+				default: "hostname"
+				examples: ["host"]
+				syntax: "literal"
+			}
+		}
+		msgid_key: {
+			common:      false
+			description: "The key for the field that will be used as the Syslog `MSGID`."
+			required:    false
+			warnings: []
+			type: string: {
+				default: "msgid"
+				examples: ["msg_is"]
+				syntax: "literal"
+			}
+		}
+		procid_key: {
+			common:      false
+			description: "The key for the field that will be used as the Syslog `PROCID`."
+			required:    false
+			warnings: []
+			type: string: {
+				default: "procid"
+				examples: ["proc_id"]
+				syntax: "literal"
+			}
+		}
+		severity_key: {
+			common: false
+			description: """
+				The key for the field that will be used as the Syslog severity. The following value will be recognized as
+				valid severity: integer from 0 (included) to 7 (included) and the matching string representation: `emerg`,
+				`alert`, `crit`, `err`, `warning`, `notice`, `info` and `debug`. Please check the
+				[RFC5424](https://tools.ietf.org/html/rfc5424) for additional information.
+				"""
+			required: false
+			warnings: []
+			type: string: {
+				default: "severity"
+				examples: ["sev"]
+				syntax: "literal"
+			}
+		}
+		default_facility: {
+			common: false
+			description: """
+				The default facility when there is no facility in the message or when it cannot be matched to a known
+				facility. Possible values and their descriptions comes from the [RFC5424](https://tools.ietf.org/html/rfc5424.
+				"""
+			required: false
+			warnings: []
+			type: string: {
+				default: "syslog"
```

Comment:
I don't know that `syslog` is the best default here as that seems to indicate that the messages are coming from the syslog daemon itself. I might default to `user`.

Reply:
I agree, I did not really think about which one to use as default. `user` seems indeed better.